torch_xla/csrc/helpers.cpp

```diff
@@ -525,13 +524,75 @@ std::pair<xla::XlaOp, xla::XlaOp> XlaHelpers::PromoteSecondValue(
                    op1, ConvertTo(op2, type2, type1, /*device=*/nullptr));
 }
 
+namespace {
+
+// Polulate 'dimensions' and 'dynamic_dimension' respectively with `count`
+// number of dimension sizes and dynamic dimensions from 'shape'.
```

Comment:
"Populates from left-most dimensions respective to count" is that an accurate statement?

Reply:
yes, done.